src/utils.js

```diff
@@ -141,6 +141,31 @@ export const reactHTMLTags = [
 
 
 /**
+ * List of "void element" tags
+ * lifted from React source
+ * https://github.com/facebook/react/blob/6a659606415d4853026df6e6d525274fdc7d35ea/src/renderers/dom/shared/ReactDOMComponent.js#L443
+ */
+export const voidElementTags = {
```

Comment:
The more I think about it the more I'm not sure this is worth it, but I wonder if there is a way we could merge this with `reactHTMLTags`:

const reactHTMLTags = {
   'a', true
   'br', false
   ...
}

And then when we make `reactConverters`, include a `selfClosed` flag in addition to the type.

Something about having each converter including all the information we need, as opposed to an external blacklist, feels more correct, but I would want to defer to @diffcunha on this one.